src/examples/server/sseAndStreamableHttpCompatibleServer.ts

```diff
@@ -26,12 +26,20 @@ const getServer = () => {
   }, { capabilities: { logging: {} } });
 
   // Register a simple tool that sends notifications over time
-  server.tool(
+  server.registerTool(
     'start-notification-stream',
-    'Starts sending periodic notifications for testing resumability',
     {
-      interval: z.number().describe('Interval in milliseconds between notifications').default(100),
-      count: z.number().describe('Number of notifications to send (0 for 100)').default(50),
+      title: 'Start Notification Stream',
+      description: 'Starts sending periodic notifications for testing resumability',
+      inputSchema: {
+        interval: z.number().describe('Interval in milliseconds between notifications').default(100),
+        count: z.number().describe('Number of notifications to send (0 for 100)').default(50),
+      },
+      outputSchema: {
+        started: z.boolean(),
+        interval: z.number(),
+        totalNotifications: z.number()
+      }
```

Comment:
looks like we're adding structured content / output schema in a lot of places - how come? Are we suggesting it should be the default to have structured output?

Reply:
Fair enough!